src/services/users.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Users = require('../db/models/users');
+
+function normalizeUser({ id, name }) {
+  return {
+    id, name,
+  };
+}
+
+async function getAll() {
+  const users = await Users.findAll();
+
+  return users;
+}
+
+async function getUserById(userId) {
+  const user = await Users.findByPk(+userId);
+
+  return user;
+}
+
+async function createNewUser(name) {
+  if (typeof name !== 'string') {
+    throw new Error('The name must be a string');
+  }
+
+  await Users.create({ name });
+}
+
+async function deleteUser(userId) {
+  await Users.destroy({
```

Comment:
Consider using parseInt with the radix parameter to safely convert userId to a number: parseInt(userId, 10)

or use Number()

https://mate-academy.github.io/style-guides/javascript.html#223-numbers-usenumberfor-type-casting-andparseintalways-with-a-radix-for-parsing-strings